contents/javascript/Javascript-framework-Angular.md

```diff
@@ -0,0 +1,226 @@
+<frontmatter>
+  title: Introduction to Angular
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Angular
+
+**Author: [Ronak Lakhotia](https://github.com/RonakLakhotia)**                                                              
+Reviewers: [Tan Heng Yeow](https://github.com/tanhengyeow), [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+## What is Angular?
+
+It's [official website](https://angular.io/docs) describes Angular as follows:
+
+>Angular is a platform that makes it easy to build applications with the web. Angular combines declarative templates, dependency injection, end to end tooling, and integrated best practices to solve development challenges. Angular empowers developers to build applications that live on the web, mobile, or the desktop.
+
+Let us understand some key terms in that definition starting with what _declarative templates_ means.
+Consider a simple web application that displays the name of a person as shown below.
+
+<iframe src="https://stackblitz.com/edit/angular-3qy4j2?embed=1&file=src/app/app.component.ts&view=preview" style="width:85%; height:380px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe> 
+
+<box type="info">
+    The sandbox above is editable. You can enter your name in the text box and see how the UI is automatically updated. You can also view and edit the code by clicking on the Both button. 
+</box>
+
+The UI of an Angular app is defined in _templates_ which contain a mix of HTML code and HTML-like custom tags. The template for the above example (defined in `person.component.html`) is given below.
```

Comment:
I think you can standardise to using semi colon such as "given below:"

I saw you using ":" for the next paragraph :p